CacheWarmer/HydratorCacheWarmer.php

```diff
@@ -66,4 +70,21 @@ public function warmUp($cacheDir)
             $dm->getHydratorFactory()->generateHydratorClasses($classes);
         }
     }
+
+    private function registerTypes() : void
+    {
+        if (! $this->container->hasParameter('doctrine_mongodb.odm.types')) {
+            // no types defined
+            exit;
```

Comment:
I'm pretty sure this should be a `return`, not an `exit` ;)
```suggestion
            return;
```